drivers/Watchdog.h

```diff
@@ -52,9 +61,13 @@ namespace mbed {
 class Watchdog : private NonCopyable<Watchdog>  {
 public:
 
-    /** As Watchdog might not stop ever, there is just one instance - we use single instance.
-      * This ensures we keep Watchdog alive. To operate watchdog, use start/stop methods.
-      */
+    /** Get a reference to the single Watchdog instance in the system.
+     *
+     * There is only one watchdog peripheral and only one Watchdog instance
```

Comment:
That may not be strictly correct that there's only one instance in the system. But we explained it above so probably we don't need to repeat it here.